README.md

```diff
@@ -11,9 +11,13 @@ Use this tutorial: http://www.howtogeek.com/249966/how-to-install-and-use-the-li
 1. Install the latest Python 2.7: https://www.python.org/downloads/release/python-2712/
```

Comment:
Markdown files look cleaning with user hyperlinks as opposed to just having the URL